.github/bot-files/comments.yml

```diff
@@ -0,0 +1,13 @@
+### You can define all your comments in this file for various purposes
+# for now you'll need to difine every comment here, in the future comments will be optional.
+# please see the example for the file below
+
+prFirstTimeContributor: Many thanks for sending out the PR, Welcome to our Community
```

Comment:
We can but I am not sure as  that can be redundant, because the link to contribution docs is already present in the PR template. Let me know if we can improve this message :)

Reply:
done.